packages/appium-tizen-tv-driver/test/e2e/driver.e2e.spec.js

```diff
@@ -112,7 +112,8 @@ describe('TizenTVDriver', function () {
         port: appiumServerPort,
         connectionRetryCount: 0,
         logLevel: 'debug',
-        capabilities,
+        // @ts-ignore wdio's types are weird?
+        capabilities
```

Comment:
i have done this in another project to make TS happy w/ wdio caps. Maybe it would work here?

```ts
const WDIO_OPTS = {
  hostname: '0.0.0.0',
  path: '/',
  connectionRetryCount: 0,
  capabilities: TEST_CAPS,
} satisfies RemoteOptions;
```

Reply:
yeah that looks weird.